package.json

```diff
@@ -46,10 +45,14 @@
     "remark-preset-github": "^0.0.16",
     "rimraf": "^3.0.0",
     "should": "^13.2.3",
-    "xo": "^0.25.3"
+    "xo": "^0.25.3",
+    "ioredis": "^5.2.4"
+  },
+  "peerDependencies": {
+    "ioredis": "5.x"
   },
   "engines": {
-    "node": ">= 4"
+    "node": ">= 14"
```

Comment:
bumping node min version to 14 is not necessary, ioredis v5 states that it is still compatible with node v12.
https://github.com/luin/ioredis/wiki/Upgrading-from-v4-to-v5

and in its package.json it is set to >= 12.22.0
https://github.com/luin/ioredis/blob/92aefaac13f42ad14986fde47c640ff8edc415ef/package.json#L92